aspnet/hosting/http-platformhandler.rst

```diff
@@ -0,0 +1,53 @@
+.. _http-platformhandler:
+
+HTTP PlatformHandler
+====================
+
+By `Sourabh Shirhatti`
+
+In ASP.NET 5 the web application is hosted by an external process outside of IIS. The HTTP PlatformHandler is an IIS 8.0+ module which is responsible for process management of http listeners and to proxy requests to processes that it manages. This document provides an overview of how to configure the HTTP PlatformHandler module for shared ASP.NET 5 hosting.
+
+Installing the HTTP PlatformHandler
+-----------------------------------
+
+To get started with hosting with ASP.NET 5 applications you will need to install the HTTP PlatformHandler version 1.2 or higher on an IIS 8 or higher server. Download links are below
+
+* `64 bit HTTP PlatformHandler <http://go.microsoft.com/fwlink/?LinkID=690721>`_ 
+* `32 bit HTTP PlatformHandler <http://go.microsoft.com/fwlink/?LinkId=690722>`_ 
+
+
+Configuring the HTTP PlatformHandler
```

Comment:
Is unlocking the handlers section of web.config relevant here, too? Merely installing the module won't do it for you (so far). If ASP.NET isn't installed on the server, that section will not be unlocked automatically; however, there is talk of changing this for an upcoming release of the next MSI. https://github.com/aspnet/IISIntegration/issues/24